### ISSUE TITLE
[iOS][a11y] Investigate accessibilityActivationPoint in UIAccessibility

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#187792]
### Use case

According to https://developer.apple.com/videos/play/wwdc2026/220/

SwiftUI has an API to listen to double tap and hold, they call it passthrough gesture.
```swift
import SwiftUI

struct CoffeeDispenserView: View {
    @State var coffee: Double = 0.0

    var body: some View {
        CoffeeSlider(value: coffee)
            .accessibilityActivationPoint(
                UnitPoint(x: 0.5, y: 1 - coffee)
            )
    }
}
```
Currently flutter doesn't have similar API to listen for double tap and hold in voiceover

### Proposal

The code in the video is using swiftUI, but there is an UIKit API https://developer.apple.com/documentation/objectivec/nsobject-swift.class/accessibilityactivationpoint?language=objc

We need to investigate whether iOS sends double tap and hold to this API, if so how can we support this API in flutter.
